website/docs/plugin/testing/acceptance-tests/known-value-checks/not-null.mdx

```diff
@@ -0,0 +1,40 @@
+---
+page_title: 'Plugin Development - Acceptance Testing: Known Values'
+description: >-
+    NotNull Value Checks for use with Plan Checks or State Checks.
+---
+
+# NotNull Known Value Checks
+
+The known value checks that are available for values that are not null are:
+
+* [NotNull](/terraform/plugin/testing/acceptance-tests/known-value-checks/null#notnull-check)
+
+## `NotNull` Check
+
+The [NotNull](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/knownvalue#NotNull) check tests that a resource attribute, or output value is not null.
```

Comment:
Nit: I wonder if its worth mentioning here or anywhere else that "not null" also means "any known value"?

Reply:
I've update the docs here. I didn't modify the Go docs for `knownvalue.NotNull`, as it seems explicit that the check is evaluating whether the passed value is nil or not.